Parsers/ASimProcessEvent/Parsers/ASimProcessCreateTrendMicroVisionOne.yaml

```diff
@@ -0,0 +1,156 @@
+    https://automation.trendmicro.com/xdr/api-v3#tag/Observed-Attack-Techniques-Pipeline/paths/~1v3.0~1oat~1dataPipelines~1%7Bid%7D~1packages~1%7BpackageId%7D/get
+    https://automation.trendmicro.com/xdr/api-v3#tag/Observed-Attack-Techniques/paths/~1v3.0~1oat~1detections/get
+Description: |
+  This ASIM parser supports normalizing Trend Micro Vision One logs to the ASIM Process Create normalized schema. Trend Micro Vision One events are captured through Trend Vision One data connector which ingests XDR logs into Microsoft Sentinel through the Trend Vision One API.
+ParserName: ASimProcessCreateTrendMicroVisionOne
+EquivalentBuiltInParser: _ASim_ProcessEvent_CreateTrendMicroVisionOne
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let GetFilenamePart = (path: string) { tostring(split(path, @'\')[-1]) };
+  let EventSeverityLookup = datatable(detail_filterRiskLevel_s: string, EventSeverity: string)[
+      "low", "Low",
+      "medium", "Medium",
+      "high", "High",
+      "info", "Informational",
+      "critical", "High"
+  ];
+  let IntegrityLevelLookup = datatable(IntegrityLevel: real, IntegrityType: string)
+      [
+      0, "Untrusted",
+      4096, "Low",
+      8192, "Medium",
+      12288, "High",
+      16384, "System"
+  ];
+  let parser = (disabled: bool=false) {
+      TrendMicro_XDR_OAT_CL
+      | where not(disabled)
+          and detail_eventId_s == "TELEMETRY_PROCESS"
+          and detail_eventSubId_s == "TELEMETRY_PROCESS_CREATE"
+      | parse filters_s with * "[" filters: string "]"
+      | parse-kv filters as (description: string, name: string) with (pair_delimiter=",", kv_delimiter=":", quote='"')
+      | extend
+          ActingProcessId = tostring(toint(detail_processPid_d)),
+          TargetProcessId = tostring(toint(detail_objectPid_d)),
+          ParentProcessId = tostring(toint(detail_parentPid_d)),
+          TargetProcessCreationTime = unixtime_milliseconds_todatetime(detail_objectLaunchTime_d),
+          ActingProcessCreationTime = unixtime_milliseconds_todatetime(detail_processLaunchTime_d),
+          ActingProcessFilename = GetFilenamePart(detail_processFilePath_s),
+          ParentProcessCreationTime = unixtime_milliseconds_todatetime(detail_parentLaunchTime_d),
+          ParentProcessName = detail_parentName_s,
+          TargetProcessFilename = GetFilenamePart(detail_objectFilePath_s),
+          ActingProcessFileSize = tolong(detail_processFileSize_d),
+          TargetUserSessionId = tostring(toint(detail_objectAuthId_d)),
+          ActorSessionId = tostring(toint(detail_authId_d)),
+          TargetProcessMD5 = replace_string(detail_objectFileHashMd5_g, "-", ""),
+          ActingProcessMD5 = replace_string(detail_processFileHashMd5_g, "-", ""),
+          ParentProcessMD5 = replace_string(detail_parentFileHashMd5_g, "-", ""),
+          AdditionalFields = bag_pack(
+                        "description",
+                        description,
```

Comment:
can we map description with EventMessage?

Reply:
Done